### PR TITLE
TCK Challenge #1190 : Extending the EE8TCK to run on JDK11+

### PR DESCRIPTION
**Pull request by alwin-joseph**
**Fixes Issue**
https://github.com/jakartaee/platform-tck/issues/1190

**Describe the change**
- Backported the signature test tool changes to not depend on JDK classes
- Backported the jsonp test fix for date format to run on JDK11+
- Replaced the usage of java.rmi.activation.Activator class so those tests work in JDK17
- Updated the Platform TCK version to 8.0.3, Jaxws TCK to 2.3.1

### CODE REVIEW

**Comment by scottmarlow** [comment]
Note that we don't really need lib/sigtestdev.jar but is also fine to replace it with lib/sigtest.jar

File: src/com/sun/ts/tests/jsonb/api/annotation/AnnotationTest.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2017, 2018 Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2017, 2020 Oracle and/or its affiliates. All rights reserved.
```
Comment (scottmarlow) [inline src/com/sun/ts/tests/jsonb/api/annotation/AnnotationTest.java:2]: 2020 may as well change to 2023.

File: src/com/sun/ts/tests/signaturetest/SigTestData.java [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2007, 2018 Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2007, 2021 Oracle and/or its affiliates. All rights reserved.
```
Comment (scottmarlow) [inline src/com/sun/ts/tests/signaturetest/SigTestData.java:2]: 2021 can be 2023

File: src/com/sun/ts/tests/signaturetest/SigTestDriver.java [hunk 3]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2007, 2018 Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2007, 2021 Oracle and/or its affiliates. All rights reserved.
```
Comment (scottmarlow) [inline src/com/sun/ts/tests/signaturetest/SigTestDriver.java:2]: 2021 should be 2023

File: src/com/sun/ts/tests/signaturetest/SigTestDriver.java [hunk 4]
```diff
@@ -58,7 +58,29 @@ public class SigTestDriver extends SignatureTestDriver {
   private static final String DEBUG_FLAG = "-Debug";
 
   private static final String FORMATPLAIN_FLAG = "-FormatPlain";
-
+  
+  private static final String EXCLUDE_JDK_CLASS_FLAG = "-IgnoreJDKClass";
+
+  private static String[] excludeJdkClasses = {
```
Comment (scottmarlow) [inline src/com/sun/ts/tests/signaturetest/SigTestDriver.java:64]: :+1:

File: src/com/sun/ts/tests/signaturetest/saaj/SAAJSigTest.java [hunk 5]
```diff
@@ -123,14 +123,9 @@ public void setup(String[] args, Properties p) throws Fault {
...
   protected SignatureTestDriver getSigTestDriver() {
-    String version = (String) System.getProperties().get("java.version");
-    if (version.startsWith("9"))
-      driver = new Jdk9SigTestDriver();
-    if (driver == null) {
-      driver = SignatureTestDriverFactory
-          .getInstance(SignatureTestDriverFactory.SIG_TEST);
-    }
-
+    driver = SignatureTestDriverFactory
```
Comment (gurunrao) [inline src/com/sun/ts/tests/signaturetest/saaj/SAAJSigTest.java:126]: SAAJ standalone TCK Sigtest update required?

**Comment by scottmarlow** [comment]
Have we staged Platform + Standalone TCK builds yet?

File: src/com/sun/ts/tests/signaturetest/saaj/SAAJSigTest.java (hunk 5 above)
Comment (alwin-joseph) [reply src/com/sun/ts/tests/signaturetest/saaj/SAAJSigTest.java:126]: The SAAJ TCK was using Jdk9SigTestDriver which is not required and removed.

**Review by gurunrao — APPROVED** [review]
LGTM

**Comment by scottmarlow** [comment]
Thanks @alwin-joseph for the change!  Great to see that we finally have a `8.0.x` branch as well!

**Pull request merged**